app/views/qualifications/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <span class="govuk-caption-xl"><%= @user.name.possessive %></span>
+    <h1 class="govuk-heading-xl">Teaching qualifications</h1>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds-from-desktop">
+        <%= render InductionSummaryComponent.new(qualification: @induction) if @induction %>
+        <%= render QtsSummaryComponent.new(qualification: @qts) if @qts %>
+        <%= render IttSummaryComponent.new(qualification: @itt) if @itt %>
+      </div>
+
+      <div class="govuk-grid-column-one-third app__details-menu">
+        <h3 class="govuk-heading-m">Your details</h3>
+        <p>
+          Name on certificates<br /> 
+          <strong><%= @user.name.full %></strong>
+        </p>
+
+        <p>
+          Date of birth<br /> 
+          <strong><%= @user.date_of_birth.to_fs(:long_uk) %></strong>  
+        </p>
+
+        <p>
+          Teacher Reference number (TRN)<br /> 
+          <strong><%= @user.trn %></strong>
```

Comment:
One thing to note here — looking at the [language used in the prototype](https://qualifications-prototype.herokuapp.com/landing-pages/v2/account-details), I think this info comes from the `/v3/teacher` API call rather than from the User object (which caches details from Identity).

@gpeng — does that sound about right? I can card up updating this if so.

Reply:
Yes. I think so. This should be from DQT